wagtail_airtable/views.py

```diff
@@ -58,30 +58,60 @@ def get_validated_models(self):
             ]
         """
         airtable_settings = getattr(settings, "AIRTABLE_IMPORT_SETTINGS", {})
-        validated_models = []
 
+        # Loop through all the models in the settings and create a new dict
+        # of the unique settings for each model label.
+        # If settings were used more than once the second (3rd, 4th, etc) common settings
+        # will be bulked into a "grouped_models" list.
+        tracked_settings = []
+        models = {}
         for label, model_settings in airtable_settings.items():
+            if model_settings not in tracked_settings:
+                tracked_settings.append(model_settings)
+                models[label] = model_settings
+                models[label]['grouped_models'] = []
+            else:
+                for label2, model_settings2 in models.items():
+                    if model_settings is model_settings2:
+                        models[label2]['grouped_models'].append(label)
+
+        # Validated models are models that actually exist.
+        # This way fake models can't be added.
+        validated_models = []
+        for label, model_settings in models.items():
+            # If this model is allowed to be imported. Default is True.
             if model_settings.get("AIRTABLE_IMPORT_ALLOWED", True):
+                # A temporary variable for holding grouped model names.
+                # This is added to the validated_models item later.
+                # This is only used for displaying model names in the import template
+                _grouped_models = []
+                # Loop through the grouped_models list in each setting, validate each model,
+                # then add it to the larger grouped_models
+                if model_settings.get("grouped_models"):
+                    for grouped_model_label in model_settings.get("grouped_models"):
+                        if "." in grouped_model_label:
+                            try:
+                                model = get_model_for_path(grouped_model_label)
+                                _grouped_models.append(model._meta.verbose_name_plural)
+                            except ObjectDoesNotExist:
+                                raise ImproperlyConfigured(
+                                    "%r is not recognised as a model name." % label
+                                )
+
                 if "." in label:
```

Comment:
Why are these checks necessary? You're already validating in the try/except block, and tbh I think it might make more sense to have this logic inside the `get_model_for_path` function

Reply:
Good idea. I put the logic in that function. It returns a model object or False. Updated tests along with it 👌